utils/documentation_tests.txt

```diff
@@ -78,9 +78,12 @@ src/transformers/models/hubert/modeling_hubert.py
 src/transformers/models/imagegpt/configuration_imagegpt.py
 src/transformers/models/layoutlm/modeling_layoutlm.py
 src/transformers/models/layoutlm/modeling_tf_layoutlm.py
+src/transformers/models/layoutlm/configuration_layoutlm.py
```

Comment:
The lines with `configuration` should go before the lines with `modeling` (within the same model) 🙏